src/ui/marker.js

```diff
@@ -5,14 +5,15 @@ import window from '../util/window.js';
 import LngLat from '../geo/lng_lat.js';
 import Point from '@mapbox/point-geometry';
 import smartWrap from '../util/smart_wrap.js';
-import {bindAll, extend} from '../util/util.js';
+import {bindAll, extend, radToDeg} from '../util/util.js';
 import {type Anchor, anchorTranslate} from './anchor.js';
 import {Event, Evented} from '../util/evented.js';
 import type Map from './map.js';
 import type Popup from './popup.js';
 import type {LngLatLike} from "../geo/lng_lat.js";
 import type {MapMouseEvent, MapTouchEvent} from './events.js';
 import type {PointLike} from '@mapbox/point-geometry';
+import {tiltAt, centerToScreen} from '../geo/projection/globe_util.js';
```

Comment:
Should these follow the convention of other functions in that file, and start with `globe*`. I also think `tiltAt` could be more description and `centerToScreen` refer to `point`, so it'd be:
- `tiltAt` -> `globeTiltAtPoint`
- `centerToScreen` -> `globeCenterToPoint`

Reply:
Good idea Karim!

Reply:
I think that "Point" can be confusing as to whether it's referring to screen or map coordinates. What do you think about `globeTiltAtScreenPoint` and `globeCenterToScreenPoint`?

Reply:
That works, thanks for the update.